<launch>
  <!-- image transport from compressed to raw -->
  <node pkg="image_transport" type="republish" name="republish" args="compressed in:=camera/image raw out:=camera/image" />

  <!-- image rectification -->
  <node ns="camera" pkg="image_proc" type="image_proc" name="image_proc" args="_approximate_sync:=true" output="screen">
    <remap from="image_raw" to="image" />
    <param name="queue_size" value="20" />
  </node>
</launch>
